packages/webapp/src/components/CropCatalogue/CropStatusInfoBox/styles.module.scss

```diff
@@ -0,0 +1,19 @@
+.container {
```

Comment:
License please!

Reply:
Dang we do that in styles? boo haha.. will add

Reply:
k its in .. also added to the other preexisting scss file.

Reply:
hold on now i see cropstatus doesnt have it either...

Reply:
k all touched files updated.